stdlib/jdbc/src/main/java/org/ballerinax/jdbc/datasource/SQLDatasource.java

```diff
@@ -60,8 +61,12 @@ public SQLDatasource init(SQLDatasourceParams sqlDatasourceParams) {
         } catch (PanickingDatabaseException e) {
             throw ErrorGenerator.getSQLDatabaseError(e);
         }
+
         try (Connection con = getSQLConnection()) {
+
```

Comment:
I feel like it's better to remove these newlines. Since the content is bound to along with the try block?

Reply:
Thanks. Removed extra new lines.